### PR TITLE
0.19.11 release 

### PR DESCRIPTION
**Pull request by youknowone**
note that this PR is targetting `0.19` branch, not the master

One of suggestion for #778

### CODE REVIEW

**Comment by youknowone** [comment]
oops, It must be 0.19.11

**Comment by youknowone** [comment]
reset 0.19 branch to tag 0.19.9 and rebased patch onto it.

File: lalrpop-util/Cargo.toml [hunk 1]
```diff
@@ -3,13 +3,13 @@ name = "lalrpop-util"
...
 license = "Apache-2.0 OR MIT"
-version = "0.19.9" # LALRPOP
+version = "0.19.11" # LALRPOP
 authors = ["Niko Matsakis <niko@alum.mit.edu>"]
 workspace = ".."
 edition = "2018"
 
 [dependencies]
-regex = { version = "1", optional = true }
+regex = { version = "1", features = ["std", "unicode"], optional = true }
```
Comment (yannham) [inline lalrpop-util/Cargo.toml:12]: Should we apply the discipline proposed in https://github.com/lalrpop/lalrpop/issues/775#issuecomment-1523312554? That is, enable those features at the level of the workspace, and don't enable any feature at the crate-level to be sure that they won't get messed by a specific crate of the LALRPOP workspace changing configuration?

File: lalrpop/Cargo.toml [hunk 2]
```diff
@@ -23,8 +23,8 @@ ena = { version = "0.14", default_features = false }
 is-terminal = "0.4.2"
 itertools = { version = "0.10", default_features = false, features = ["use_std"] }
 petgraph = { version = "0.6", default_features = false }
-regex = { version = "1", default_features = false, features = ["std"] }
-regex-syntax = { version = "0.6", default_features = false }
+regex = { version = "1", default_features = false, features = ["std", "unicode"] }
```
Comment (yannham) [inline lalrpop/Cargo.toml:26]: As an aside, we seem to enable exactly the same features anyway

File: lalrpop-util/Cargo.toml (hunk 1 above)
Comment (Pat-Lafon) [reply lalrpop-util/Cargo.toml:12]: Did you see #783? I was realizing that I'm not sure how these different levels of feature enabling intersect with lalrpop's features. Of course I'm all for turning everything on, but I just want to make sure that people who use lalrpop without the lexer still get whatever minimal setup they are looking for.

**Comment by ithinuel** [comment]
I did some exploration & testing trying to fix lalrpop tests' builds when not building with `all-features` and as far as I (now) understand it, the point is that the extra features required on the regex and regex-syntax module are dependent on the grammar the user is building.

So enabling the whole "unicode" feature may not be where lalrpop want to aim (or at least not where it used to aim at).

#783 provides a set of fixes and an update of the CI script to capure similar feature related future breakage.

File: lalrpop-util/Cargo.toml (hunk 1 above)
Comment (youknowone) [reply lalrpop-util/Cargo.toml]: I dropped this part and applied #783

**Pull request merged**